istio-model/src/test/java/me/snowdrop/istio/template/quota/QuotaTest.java

```diff
@@ -53,12 +53,12 @@ public void quotaRoundtripShouldWork() throws Exception {
                 .withName("requestcount")
                 .withNamespace("istio-config-default")
                 .endMetadata()
-                .withNewQuotaSpec()
+                .withNewQuotaQuotaSpec()
```

Comment:
This is the weird builder name. @iocanel any idea on how we could improve this situation?

Reply:
Such a solution would be good, indeed, @iocanel